app/views/layouts/application.html.erb

```diff
@@ -10,8 +10,12 @@
   <body>
     <%= render "layouts/header" %>
     <div class="container">
+      <% flash.each do |message_type, message| %>
+        <div class="alert alert-<%= message_type %>"><%= message %></div>
```

Comment:
please using Rails helper `content_tag` to rewrite this line. Don't use plain HTML code